src/consensus/params.h

```diff
@@ -22,22 +22,31 @@ enum DeploymentPos
 /**
  * Struct for each individual consensus rule change using BIP9.
  */
-struct BIP9Deployment {
+struct VBitsDeployment {
     /** Bit position to select the particular bit in nVersion. */
     int bit;
-    /** Start MedianTime for version bits miner confirmation. Can be a date in the past */
-    int64_t nStartTime;
-    /** Timeout/expiry MedianTime for the deployment attempt. */
-    int64_t nTimeout;
+    /** Start block height for version bits miner confirmation. Must be a retarget block, can be in the past. */
+    int startheight;
+    /** Timeout/expiry block height for the deployment attempt. Must be a retarget block. */
+    int timeoutheight;
+    /** Threshold for activation */
+    int threshold;
+    /**
+     * If lock in occurs, delay activation until at least this block height. Activations only occur on retargets.
+     */
+    int64_t m_min_activation_height;
```

Comment:
`{0}` here can avoid needing to initialise it everywhere it isn't used (and reduce the risk of ever forgetting to do so)

Reply:
Done